### PR TITLE
Change version to 1.3.0dev

### PR DESCRIPTION
**Pull request by sunank200**
# Description
## What is the current behavior?

Version in main is 1.2.0

## What is the new behavior?

- Change version to 1.3.0dev

## Does this introduce a breaking change?
No

### Checklist
- [x] Created tests which fail without the change (if possible)
- [x] Extended the README / documentation, if necessary

### CODE REVIEW

**Review by pankajastro — APPROVED** [review]

File: python-sdk/src/astro/__init__.py [hunk 1]
```diff
@@ -1,6 +1,6 @@
 """A decorator that allows users to run SQL queries natively in Airflow."""
 
-__version__ = "1.2.0"
+__version__ = "1.3.0dev"
```
Comment (pankajastro) [inline python-sdk/src/astro/__init__.py:3]: it should be 1.3.0 or 1.2.1?

Comment (kaxil) [reply python-sdk/src/astro/__init__.py:3]: 1.3.0 -- Main should always point to the next minor

Comment (kaxil) [inline python-sdk/src/astro/__init__.py:3]:
```suggestion
__version__ = "1.3.0.dev1"
```

Comment (kaxil) [inline python-sdk/src/astro/__init__.py:3]:
```suggestion
__version__ = "1.3.0.dev1"
```

**Review by kaxil — APPROVED** [review]

**Pull request merged**